OrbitAsm/OrbitAsmC.h

```diff
@@ -1,3 +1,9 @@
+/*
+ * Copyright (c) 2020 The Orbit Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+
```

Comment:
so.. I suspect /* */ here are because of "it could be a c header"? Doesn't  new C standard recognize // ? If I am not mistaken starting with C99?

Reply:
Yes, it's purely a tools issue. I think we actually need to use /* */ everywhere so this is all recognized correctly.